README.md

```diff
@@ -124,7 +124,7 @@ export CUDA_HOME=$CONDA_PREFIX
 export CUDA_HOME="/usr/local/cuda"
 ```
 
-Fianlly, install this project by:
+Fianlly, install `align-anything` by:

```

Comment:
Maybe we can unify it to Align-Anything

Reply:
from xuehai: the name of the module is best in lowercase.

Reply:
unify to align-anything

Reply:
Nice advice ! But conda env name is also `align-anything`, so maybe `align-anything` is better.

Reply:
Thank you for the quick response! I think this has addressed most of my concerns.